ibis/omniscidb/client.py

```diff
@@ -392,6 +395,41 @@ def load_data(self, df):
         stmt = ddl.LoadData(self._qualified_name, df)
         return self._execute(stmt)
 
+    def read_csv(
+        self,
+        path: Union[str, Path],
+        header: bool = True,
+        quotechar: str = '"',
+        delimiter: str = ',',
+    ):
+        """
+        Load data into an Omniscidb table from CSV file.
+
+        Wraps the COPY FROM DML statement.
+
+        Parameters
+        ----------
+        path : str or pathlib.Path
+          Path to the input data file
+        header : bool, optional, default True
+          Indicating whether the input file has a header line
+        quotechar : str, optional, default '"'
+          The character used to denote the start and end of a quoted item.
+        delimiter : str, optional, default ','
+
+        Returns
+        -------
+        query : OmniSciDBQuery
+        """
+        kwargs = {
+            'header': header,
+            'quote': quotechar,
+            'quoted': True if quotechar != '' else False,
```

Comment:
maybe you can just replace it by `'quoted': bool(quotechar),`

Reply:
done